.gitignore

```diff
@@ -2,4 +2,23 @@
 .DS_Store
 deployments/current
 node_modules
-.env
\ No newline at end of file
+.env
+
+# Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+.terraform.lock.hcl
+
```

Comment:
ignoring these files since this is a shared library.  readme includes documentation about configuring state managment.